homeassistant/components/namecheapdns/__init__.py

```diff
@@ -15,37 +15,50 @@
 
 DOMAIN = 'namecheapdns'
 
+CONF_HOSTS = CONF_HOST + 's'  #FIXME Put in homeassistant.const ?
```

Comment:
Use this from `homeassistant.const` instead: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py#L87